packages/next/server/next-dev-server.ts

```diff
@@ -85,6 +85,16 @@ export default class DevServer extends Server {
       {
         maxRetries: 0,
         numWorkers: this.nextConfig.experimental.cpus,
+        forkOptions: {
+          env: {
+            ...process.env,
+            // discard process.env.NODE_OPTIONS --inspect flag otherwise two debuggers are started in inspect
+            // mode when users will try to debug their Next.js application with NODE_OPTIONS='--inspect' next dev
+            NODE_OPTIONS: process.env.NODE_OPTIONS
+              ? process.env.NODE_OPTIONS.replace('--inspect', '')
```

Comment:
Fixed here: https://github.com/vvo/next.js/pull/1

Reply:
Note: Downgrading to v8.1.0 fixed the issue

Reply:
Overriding `process.env.NODE_OPTIONS` seems to do the job:
https://github.com/aleclarson/next.js/commit/0248a44e620f499271e8ccb43c69138f8d010422

Apart from "purity", is there any reason this approach should be avoided?

Reply:
> Apart from "purity", is there any reason this approach should be avoided?

I think its fine, the feature of debugging a Next.js application for end users outweights the "purity" question for me. Sorry I have no time left to properly fix this but we know where the bug is at least, if someone from Next.js jumps in we'll see

Reply:
Even with my fix, I'm still unable to set breakpoints in my `pages/api` functions. :(